src/test_pg_1995.jl

```diff
@@ -0,0 +1,131 @@
+using Plots
```

Comment:
This test is great! :) I have a few points:
- I've run it locally and the theta profiles for the RSL look somewhat off (negative and very different to the no-RSL case). The u profiles look fine - at least from a  qualitative comparison with Fig. 6.8 in Bonan's textbook.
- While it's nice to have the plot test somewhere, shall we add some automated value checking test for CI? 
- I presume the plan is to move this test from `/src`?

Reply:
Yes - plan is to move this into `test` and have more quantitative measures (with generated profiles for example)